docs/data_providers/data_format/ncbi_taxa.md

```diff
@@ -89,20 +83,20 @@ if you want to use `Crbe` in your data worksheets, rather than typing out
   headers in the worksheet, with the exception of `clade` and `no rank` as these ranks
   can be duplicated within a taxon hierarchy.
   
-  You are only required to provide the taxonomic name for the specific rank that you are
-  trying to match and can leave other fields empty. However it is probably more useful
-  to provide a more complete taxonomy! If you do provide higher taxonomic information
-  then it must be congruent with the hierarchy for the focal taxon. For example,
-  specifying Family Anatidae is sufficient to identify waterfowl, but providing Order
-  Carnivora (rather than Order Anseriformes) would result in an error.
-
-  Names can be provided in plain text, or alternatively in a commonly used notation,
+    You are only required to provide the taxonomic name for the specific rank that you
```

Comment:
Leading spaces at at paragraph starts here and below?

Reply:
Oh this is needed so that the paragraph gets treated as a continuation of the bullet point rather than a new paragraph outside the bullet point. It's not the best notation but I can't see an obvious work around